plugins/task/sessiongc/services/provider.php

```diff
@@ -27,19 +27,21 @@
      *
      * @return  void
      *
-     * @since   4.4.0
+     * @since   4.2.0
```

Comment:
```suggestion
     * @since   __DEPLOY_VERSION__
```
As the file has been moved from an old to a new plugin, I think we should use `__DEPLOY_VERSION__` here. It seems it was done like that when the plugin has been refactored to the new structure, that_s where the 4.4.0 comes from, I assume.

But I'm not 100% sure more opinions would be welcome.